recbole/evaluator/collector.py

```diff
@@ -190,7 +194,7 @@ def eval_batch_collect(
         if self.register.need("data.label"):
             self.label_field = self.config["LABEL_FIELD"]
             self.data_struct.update_tensor(
-                "data.label", interaction[self.label_field].to(self.device)
```

Comment:
Redundant conversion because all input will be transferred to `cpu` in the `update_tensor` method.